src/main/scala/exercises/06_Collections.scala

```diff
@@ -114,7 +114,10 @@ object Collections {
    * pattern match or some other method.
    */
   def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
+    iList match {
```

Comment:
I think this function is supposed to consider the value of the `Int`s rather than its position.

For example:
```scala
oddElements(List(2,42, 47)) == List(47)